LayoutTests/fast/svg/foreign-object-without-svgroot-crash.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script>
+function testFunction() {
```

Comment:
Can we just name this `test()`?

Reply:
I‘d go for something descriptive: both test / testFunction are too generic. „reparentForeignObject“ / „replaceBodyWithForeignObject“?

Furthermore, I would move the test in a LayoutTests/svg/xxx subdirectory.